knowledge/Users/klawrenc/.local/share/instructlab/taxonomy/knowledge/films/history/attribution.txt

```diff
@@ -0,0 +1,5 @@
+Title of work: History of films
+Link to work: https://www.blackwellpublishing.com/content/BPL_Images/Content_store/ThumbNail/BURGOYNE9781405146029/9781405146029.pdf
```

Comment:
Unfortunately, we cannot accept this source due to licensing restrictions. To keep InstructLab open source, we have to vet all sources to ensure they are also open source. To learn more, read our current [licensing limitations for sources](https://github.com/instructlab/community/blob/main/docs/README.md#license-limitations) and [list of accepted sources](https://github.com/instructlab/dev-docs/blob/main/docs/knowledge-submissions-past-wikipedia.md).